tests/integration/test_install_basic.py

```diff
@@ -316,13 +316,10 @@ def test_skip_requirements_when_pipfile(PipenvInstance):
             contents = """
 [packages]
 six = "*"
-fake_package = "<0.12"
             """.strip()
             f.write(contents)
         c = p.pipenv("install")
         assert c.returncode == 0
-        assert "fake_package" in p.pipfile["packages"]
```

Comment:
Fake package has been problematic in many different refactors, and it isn't relevant to this test case anyway.   I've made an effort to replace those other problem usages with a different relevant package for testing as well.